src/app/globals.css

```
@@ -94,8 +108,301 @@
+    background: rgba(47, 51, 61, 0.82);
+    border-color: rgba(255, 255, 255, 0.08);
+    color: #b8b1c2;
+    box-shadow: inset 0 1px 0 rgba(255, 255, 255, 0.04);
+  }
+
+  .status-pill::before {
+    content: "";
+    width: 7px;
+    height: 7px;
+    border-radius: 999px;
+    background: currentColor;
```

Stylelint (17.11.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)